Lakka v4.x general

GavinDarkglider opened the pull request:
This PR does 2 general things....

It downgrades ppsspp to version from lakka v4.2, which actually loads/works.

It bumps Lakka version to 4.3

GavinDarkglider committed:
Lakka: Bump version to 4.3

spleen1981 commented:
Downgrading ppsspp should not be necessary with #1811, if it is confirmed that part should be removed

GavinDarkglider commented:
> Downgrading ppsspp should not be necessary with #1811, if it is confirmed that part should be removed

I am less concerned about fixing newer version of core, and more concerned about not breaking already supported cores...... if this your fix fixes it on my boards, great, if not, I would prefer to keep the working versions in stable tree.

Ntemis commented:
@GavinDarkglider ppsspp is now fixed thanks to @spleen1981 please revert ppsspp changes so i can merge, thanks

GavinDarkglider commented:
> @GavinDarkglider ppsspp is now fixed thanks to @spleen1981 please revert ppsspp changes so i can merge, thanks
I dropped my ppsspp changes, so this is just a version bump now.
